### ISSUE TITLE
Metakey shortcuts will send the letter into focused window

### ISSUE THREAD

**Issue by dmnmsc** [phil294/AHK_X11#38]
Hi,

I'm using gnome desktop and I have a problem with keyboard shortcuts.  Take this script as example:

```
#t::
IfWinExist, Telegram
	{
	WinActivate
	}
else
	{
	Run, telegram-desktop
	WinActivate
	}
return
```

This will send and type the letter t into the focused window.
Desired scenario: no letter t appearing in focused window. This seems to happen when you are using just meta as a key.

**Comment by phil294** [comment]
Hi!

I cannot reproduce this. Just to make sure I understand correctly: Telegram launches/activates properly, but it also does not prevent the t from being typed into the currently active window?

So a script like
```ahk
#t::Return
```
just basically doesn't work because `t` is not captured?

Are you using version 0.5.1?

**Comment by dmnmsc** [comment]
Hey!

That's it.  Telegram launches/activates properly, but it also does not prevent the t from being typed into the currently active window. Even your basic script will type the letter t into the currently active window.

[Grabación de pantalla desde 2023-03-17 18-33-39.webm](https://user-images.githubusercontent.com/8472142/225978827-17a46176-5689-4ef5-9789-7bc25385b4a7.webm)


> Is this behavior affected by the caps lock key?
Yes it is. 

> Are you using version 0.5.1?
Yes! Downloaded the new version yesterday.

**Comment by phil294** [comment]
can you please run the `#t::Return` script in console and see if it outputs (besides other stuff) a message like

    BadAccess (attempt to access private resource denied) (You can probably ignore this error)

?

**Comment by dmnmsc** [comment]
> can you please run the `#t::Return` script in console and see if it outputs (besides other stuff) a message like

```
~/Documentos $ ./ahk_x11 t.ahk 

(process:2697): Gtk-WARNING **: 12:31:54.816: Locale not supported by C library.
	Using the fallback 'C' locale.
Gtk-Message: 12:31:54.901: Failed to load module "canberra-gtk-module"
Gtk-Message: 12:31:54.903: Failed to load module "canberra-gtk-module"
Interactive AHK_X11 console (REPL). Type any command. Multi-line text not supported. Press {CTRL}+C to exit.
ahk_x11> 
```

I renamed the new version to ahk_x11. But as I said I'm using 0.5.1.

**Comment by dmnmsc** [comment]
Switching to right super key in keyboard tweaks settings solves the issue.

![image](https://user-images.githubusercontent.com/8472142/226111811-974d7e5c-b6cf-4552-9b65-9fb4f2ca2d46.png)

But then super key won't open activities/menu - so you need to install ksuperkey and set activities shortcut to ALT+F1.

It's and odd workaround. Maybe helps you to track the issue.

**Comment by phil294** [comment]
ok, thanks, I'm out of ideas for now, might quickly test it in a VM. The weird `BadAccess` error only shows when the super+t hotkey was already "grabbed" by another application such as the desktop environment's global hotkeys. And if that other program somehow sends the `k` key artificially, it cannot be prevented. Unlikely though. That's also the reason why you saw it when you ran the script twice: First one grabbed the key combo, the second one could not. But usually it really doesn't matter.

In other words: IDK, but I'll check later

**Comment by dmnmsc** [comment]
Don't worry. I can live with it thanks to the workaround. You are doing a great job. 

Just waiting the InputBox implementation to use my most desired scripts!

**Comment by phil294** [comment]
hi @dmnmsc, I have installed Ubuntu+Gnome in a VM but couldn't reproduce this. My bet is still that some other application has reserved the `T` key for you but I don't really know what's going on here. Perhaps my installation wasn't complete yet, but for now it looks like this is a wontfix because cannot reproduce.

> Just waiting the InputBox implementation to use my most desired scripts!

InputBox is now available!

**Comment by phil294** [comment]
Hi @dmnmsc, I know it's been a while but this problem might be fixed if you add `#XGrabRoot` anywhere in your script.
